sycl/test-e2e/Matrix/joint_matrix_bf16_fill_k_cache_impl.hpp

```diff
@@ -245,24 +327,30 @@ void test() {
   queue q;
   T *A = malloc_shared<T>(MATRIX_SIZE * MATRIX_SIZE, q);
   T *B = malloc_shared<T>(MATRIX_SIZE * MATRIX_SIZE, q);
+#ifdef VNNI
   T *vnniB = malloc_shared<T>(MATRIX_SIZE * MATRIX_SIZE, q);
```

Comment:
Move it to the line before matrix_vnni<T>, as it is used in a very limited context, so it makes sense to keep declaration/definition in the context, where it is used.